README.md

```diff
@@ -1,4 +1,5 @@
 # RimDev ES6 component generator
+![build status](https://ci.appveyor.com/api/projects/status/ae1taoboaxyvy3ou?svg=true)
```

Comment:
Wait, I believe I was confused with the generated **README.md** vs. this repositories **README.md**. If you got that value from AppVeyor it's probably fine. Sorry. :smiley: